merlin/datasets/testing/sequence_testing/schema.json

```diff
@@ -26,8 +26,7 @@
       "name": "item_age_days_norm",
       "type": "FLOAT",
       "valueCount": {
-        "min": "1",
-        "max": "4"
```

Comment:
Why do we need to remove the max value count from this schema and others?

Reply:
The issue is that when min and max are provided and are different, the dataloader returns a tf.SparseTensor, but now MM can only take either tf.Tensor or our convension for ragged tensors (__values and __offsets). So I remove the max to enforce it to return the ragged tensors.
I know you have already a [draft PR](https://github.com/NVIDIA-Merlin/dataloader/pull/103) that will make it return ragged tensors in such cases. As soon as it is merged we can add back the `value_count.max` from the test dataset schemas if needed.
Does that make sense?